cmd/summary.go

```diff
@@ -161,14 +83,14 @@ func GetSummaryCommand() *cobra.Command {
 							return er
 						}
 						return nil
+					} else {
+						return errors.New(fmt.Sprintf("Host %q is not supported.\nSupported hosts are: %v", specUrl.Host, supportedHosts))
 					}
 
 				} else {
-					pterm.Error.Println("Two arguments are required to compare left and right OpenAPI Specifications.")
-					return nil
+					return errors.New(fmt.Sprintf("Error parsing url: %v", err))
```

Comment:
Should we just return an error, or should I also do a `pterm.Error.println(...)`?